<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webmapAddPropertyDialog</class>
 <widget class="QDialog" name="webmapAddPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Property</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="nameEditWidget">
       <property name="placeholderText">
        <string>Name without spaces and brackets</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="descrEditWidget">
       <property name="placeholderText">
        <string>Small description about this property</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="typeComboWidget">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>string</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>file</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="listCheckWidget">
       <property name="toolTip">
        <string>Check if this property will contains multiple values</string>
       </property>
       <property name="text">
        <string>Is list?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="validValuesGroup">
       <property name="title">
        <string>Valid Values</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="validValueEditWidget"/>
          </item>
          <item>
           <widget class="QListWidget" name="validValuesListWidget"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addBtnWidget">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/symbologyAdd.png</normaloff>../icons/symbologyAdd.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delBtnWidget">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/symbologyRemove.png</normaloff>../icons/symbologyRemove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButtonWidget">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButtonWidget">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
